static/js/emoji.js

```diff
@@ -18,6 +18,20 @@ const zulip_emoji = {
     deactivated: false,
 };
 
+exports.get_emoji_url = (emoji_name) => {
```

Comment:
This function name is no good -- this function __only__ looks are realm emoji, not unicode emoji, which is confusing and given the name, and likely not something we want to export/

Reply:
Maybe what we want to do is extract a different function that returns the `emoji_data` object, and then just have the `markdown.js` caller access the data attributes of the object, and remove the direct `markdown.js` access to `emoji_codes`?  The interface problem here may be how tightly the `markdown` code digs into the emoji data structures.